spec/unit/view_helpers/display_name_spec.rb

```diff
@@ -27,7 +27,8 @@
     allow(klass).to receive(:reflect_on_all_associations).and_return [ double(name: :login) ]
     allow(subject).to receive :login
     expect(subject).to_not receive :login
-    allow(subject).to receive(:email).and_return 'foo@bar.baz'
+    allow(ActiveAdmin.application).to receive(:display_name_methods).and_return [:login, :display_name]
+    allow(subject).to receive(:display_name).and_return 'foo@bar.baz'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.